main.py

```diff
@@ -1,8 +1,21 @@
 import click
 
-@click.command()
-def hello():
-    click.echo('Hello World!')
+taskcounter = 0
 
-if __name__ == '__main__':
-    hello()
+#Create a new task
+def newtask(): 
```

Comment:
Functions should be named with a verb:

```suggestion
def create_new_task(): 
```

What to you thin?